intro.tex

```diff
@@ -12,45 +12,78 @@ \subsection{Motivation for an Early Science Program}  \label{ssec:motivation}
-The operations team may decide to first conduct a \textit{full dress rehearsal} of science operations to demonstrate team readiness prior to commencing execution of the LSST.
-In this scenario, we expect this rehearsal to take a few days to no longer than two weeks. 
-
-\item \textbf{Scenario B}:
-Prior to commencing the 10-year LSST, the operations team decides to spend up to a maximum of 2 months collecting more on-sky data to complement and extend the datasets collected during commissioning. 
-This additional data would serve the dual goals of 1) ensuring that we are fully prepared to start the 10-year survey and 2) providing the community with an exquisite early dataset to work with before DR1. 
-As per Scenario A, the operations team may decide to first conduct a \textit{full dress rehearsal} of science operations to demonstrate team readiness prior to commencing execution of the LSST. 
+\item The operational status of the observatory and progress of system integration and test activities in commisioning; 
+\item The nature and quality of the data collected during commissioning; 
+\item The readiness of the data processing pipelines and data distribution and access services. 
 \end{itemize}
-As the survey begins, all science-grade data collected during the commissioning System Optimization period and subsequent Science Validation Surveys, \S~\ref{sec:commissioning}, is reprocessed to produce the final Data Preview, DP2, which will be released 6 months following the start of operations.
-
-In both scenarios it is assumed that the Rubin Construction project delivers an integrated system that can capture, transfer and process science-grade data at the time operations begins.
-Both scenarios will include alert generation of some type, with the major distinction being the relative availability of templates in time, sky position, and filter. 
-The subset of System First Light observations that form the basis of DP1 must be taken and analysed in order to support construction completeness, meaning that DP1 is identical in each scenario.
-The DP2 data products will be the same irrespective of which scenario materializes; only the timing of the release of DP2 and the start of the 10-year survey are different between the two scenarios.
-These two scenarios presented are current as of December 2022, but are subject to change as the commissioning program is executed.
-At some future point, a single option will be adopted and executed, and at that time, the details will be more fully specified.
+Policy factors include:
+\begin{itemize}
+\item There is a 30-day embargo on all pixel data during commissioning;
+\item There is an 80-hour embargo on all pixel data throughout the entirety of Pperations, including the SV surveys and the LSST; 
+\item The construction security review, which must be successfully completed prior to the release of any Prompt data products; 
```

Comment:
Suggest to make this a complete sentence, for consistency with the other bullets.

"A construction security review must be successfully completed prior to the release of any Prompt data products;"